19/problem19.rb

```diff
@@ -0,0 +1,4 @@
+puts "Enter id:"
+id=gets.chomp!
+
+puts "ORD-%05d" % id
```

Comment:
id.rjust(count,string_to_pad) is more general function for this purpose. It give you freedom to choose string to padd also.